### PR TITLE
feat: Add Fizzbuzz.java

### PR DESCRIPTION
**Pull request by limjoonchul**
Add Fizzbuzz

### CODE REVIEW

File: src/Fizzbuzz.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+public class Fizzbuzz {
+    public static void main(String[] args) {
+        for (int i = 1; i <= 100; i++) {
+            if (i % 15 == 0){
+                System.out.println("fizzbuzz");
```
Comment (FoeverNa) [inline src/Fizzbuzz.java]: Capitalize f to F

Comment (ulgoon) [reply src/Fizzbuzz.java]: Totally agreeee!

**Review by FoeverNa — APPROVED** [review]

File: src/Fizzbuzz.java (hunk 1 above)
Comment (limjoonchul) [reply src/Fizzbuzz.java]: okay...

**Comment by FoeverNa** [comment]
Good job!

**Pull request merged**